apps/portals/template/index.html

```diff
@@ -1,9 +1,14 @@
 <!DOCTYPE html>
```

Comment:
Should the example-portal ("template") be removed?  I think we typically just use an existing portal as a starting point.

Reply:
I'm ok with removing it!